src/mux-analytics.brs

```diff
@@ -904,6 +906,34 @@ function muxAnalytics() as Object
     m._addEventToQueue(m._createEvent("rebufferend"))
   end sub
 
+  prototye.playbackModeHandler = sub(playbackMode as Object)
+    props = {}
+
+    if playbackMode.mode = Invalid
+      print "[mux-analytics] warning: playback_mode mode property not set."
+      return
+    end if
+    props.player_playback_mode = playbackMode.mode
+
+    if playbackMode.player_playback_mode_data = Invalid
+      print "[mux-analytics] warning: playback_mode player_playback_mode_data property not set."
+      return
+    end if
+
+    ' ParseJson returns invalid if provided string is not parse-able JSON
+    parsedData = ParseJson(playbackMode.player_playback_mode_data)
+    if parsedData = Invalid then
```

Comment:
Can we make `player_playback_mode_data` optional?

Like, if the input from the caller is `Invalid`, can we skip the check?

If they provide something that isn't valid JSON, we should still warn and return, but if they don't provide anything, that should be allowed.